Update Exterminate (#1187)
DFHack/scripts: changelog.txt

- `gui/launcher`: refresh default tag filter when mortal mode is toggled in `gui/control-panel` so changes to which tools autocomplete take effect immediately
- `gui/civ-alert`: you can now register multiple burrows as civilian alert safe spaces
- `exterminate`: add ``all`` target for convenient scorched earth tactics
- `exterminate`: add ``--limit`` option to limit number of exterminated creatures

> - `exterminate`: add ``--knockout`` method for a non-lethal "extermination"

Reviewer: ```suggestion
- `exterminate`: add ``knockout`` and ``traumatize`` method for a non-lethal incapacitation
```

- `gui/launcher`: refresh default tag filter when mortal mode is toggled in `gui/control-panel` so changes to which tools autocomplete take effect immediately
- `gui/civ-alert`: you can now register multiple burrows as civilian alert safe spaces
- `exterminate`: add ``all`` target for convenient scorched earth tactics

> - `exterminate`: add ``--max`` option to limit number of exterminated creatures

Reviewer: ```suggestion
- `exterminate`: add ``--limit`` option to limit number of exterminated creatures
```

- `exterminate`: add ``--knockout`` method for a non-lethal "extermination"

> - `exterminate`: add ``all`` target for convenient scorched earth tactics

Reviewer: ```suggestion
```
